packages/game-bridge/src/index.ts

```diff
@@ -170,7 +170,9 @@ window.callFunction = async (jsonData: string) => {
             crossSdkBridgeEnabled: true,
           };
           passportClient = new passport.Passport(passportConfig);
-          track(moduleName, 'initialisedPassport');
+          track(moduleName, 'initialisedPassport', {
+            time: Date.now() - markStart,
```

Comment:
hahaha, i LOVE these kind of code.  will the diff be represented well in the tracking event? 
as in, it won't confuse anyone whether it's millisec or sec or ... from segment event?